scripts/apply_def_template.py

```diff
@@ -57,7 +57,8 @@ def remove_power_pins(DEF):
         [
             "openroad",
             "-python",
-            "move_pins" f"{scriptsDir}/odbpy/defutil.py",
+            "move_pins",
```

Comment:
shouldn't `move_pins` come after ` f"{scriptsDir}/odbpy/defutil.py"`
Also I think it is `replace_pins`

Reply:
Yes and yes. Oh god...